libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -2251,6 +2290,7 @@ void AP_OSD_Screen::draw(void)
     DRAW_SETTING(heading);
     DRAW_SETTING(wind);
     DRAW_SETTING(home);
+    DRAW_SETTING(rrpm);
```

Comment:
i'm surprised this compiles with RPM disabled? It will call a non-existant function